templates/app/models/index.js

```diff
@@ -18,6 +18,18 @@ let databaseOptions = {
   dialectOptions: {}
 };
 
+<%if(config.dbDialect === 'mysql') {
+%>databaseOptions.dialectOptions.typeCast = ( field, useDefaultTypeCasting ) => {
+  if ( ( field.type === "BIT" ) && ( field.length === 1 ) ) {
+    const bytes = field.buffer();
+    return( bytes[0] === 1 );
+  }
+
+  return( useDefaultTypeCasting() );
+};
```

Comment:
[Code style] Why is there spaces in all those parenthesis ?

Reply:
FIxed